packages/smooth_app/lib/pages/onboarding/country_selector.dart

```diff
@@ -51,40 +57,29 @@ class _CountrySelectorState extends State<CountrySelector> {
           if (snapshot.connectionState != ConnectionState.done) {
             return const CircularProgressIndicator();
           }
-          return Padding(
-            padding:
-                const EdgeInsets.only(top: MEDIUM_SPACE, bottom: LARGE_SPACE),
+          return Container(
```

Comment:
why change from Padding -> Container ?

Reply:
Because a `Container` can have a `null` padding. A `Padding` can't.

Reply:
consider initializing the padding to default value of EdgeInsets.zero if not set by the caller and switching the `Container` back to `Padding`.